src/Helper/PaymentHelper.php

```diff
@@ -105,43 +113,53 @@ public function getAmountToPay(array $values, string $key): float {
   }
 
   /**
-   * Returns the secret key for NETS EasyPay.
+   * Returns the settings for the os2forms_payment module.
    *
-   * @return string
-   *   Returns the secret key for the Nets EasyPay integration.
+   * @return array<mixed>
+   *   Returns the settings for the os2forms_payment module.
    */
-  public function getSecretKey() {
-    return $this->secretKey;
+  private function getPaymentSettings() {
+    return Settings::get('os2forms_payment');
   }
 
   /**
-   * Returns the checkout key for NETS EasyPay.
+   * Returns the checkout key for the nets implementation.
    *
    * @return string
-   *   Returns the checkout key for the Nets EasyPay integration.
+   *   The Checkout key.
    */
   public function getCheckoutKey() {
-    return $this->checkoutKey;
+    return $this->getPaymentSettings()['checkout_key'];
```

Comment:
Always check that an array index exists when using it!

```suggestion
    return $this->getPaymentSettings()['checkout_key'] ?? NULL;
```